feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt

```diff
@@ -209,6 +214,41 @@ fun About(
     }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun AboutTopAppBar(
```

Comment:
I think this app bar will be fixed by this issue. 
https://github.com/DroidKaigi/conference-app-2022/issues/238

Can you add todo like

// TODO: Replace with Slot API pattern
// see: https://github.com/DroidKaigi/conference-app-2022/issues/238